cmd/dashboard/main.go

```diff
@@ -102,7 +129,14 @@ func main() {
 	controllers.StartKubeControllers(resource.K8sClient, resyncDur, dashboardConfig.installNamespace, routerHandler, ctx.Done())
 
 	logging.Log.Infof("Creating server and entering wait loop")
-	server := &http.Server{Addr: dashboardConfig.port, Handler: routerHandler}
+	CSRF := csrf.Protect(
+		getCSRFAuthKey(),
+		csrf.CookieName("token"),
+		csrf.Path("/"),
+		csrf.SameSite(csrf.SameSiteLaxMode),
+		csrf.Secure(isCSRFSecureCookieEnabled()),
+	)
+	server := &http.Server{Addr: dashboardConfig.port, Handler: CSRF(routerHandler)}
```

Comment:
Ah, so CSRF's constructor accepts a routerHandler I see, this is the key I didn't figure out, good approach

Reply:
yeah, found a load of conflicting docs scattered around the place that had me confused and sent me down the wrong path with filters etc. Eventually threw all that out and reviewed the godocs for both projects directly to figure this out. The match was not obvious at first.